README.md

```diff
@@ -21,31 +21,23 @@ Kuzzle features are accessible through a secured API. It can be used through a l
 
 # How to use it
 
-This Back-office is a static front-end application. All calls to Kuzzle are made directly in the browser. **This is why your Kuzzle Proxy IP and Websocket port (default 7512) must be accessible from everywhere if you want to use this Back-office.**
+This Admin Console is a static front-end application. All calls to Kuzzle are made directly in the browser. **This is why your Kuzzle Proxy IP and Websocket port (default 7512) must be accessible from everywhere if you want to use this Admin Console.**
 
-You can check the last version of Back-office here: [kuzzle-backoffice](http://kuzzle-backoffice.netlify.com) (or if you're using Kuzzle through SSL, you can use the version [https](https://kuzzle-backoffice.netlify.com/))
+You can check the last version of Admin Console here: [Admin Console](http://console.kuzzle.io) (or if you're using Kuzzle through SSL, you can use the [https](https://console.kuzzle.io) version)
 
-For installing the Back-office on your own host, follow next steps.
+For installing the Admin Console on your own host, follow next steps.
```

Comment:
![all your base are belong to us](https://upload.wikimedia.org/wikipedia/en/0/03/Aybabtu.png)

Moreover, I think that using the "install" verb is confusing, especially just after telling that it can be executed locally thanks to console.kuzzle.io

Proposal: `To get a working copy of the Admin Console on your computer, follow these steps:`